xorg/xorg-server.txt

```diff
@@ -0,0 +1,89 @@
+XORG-SERVER [0]
+________________________________________________________________________________
+
+The xorg-server, part of the X.Org releases, is the main component of the X 
+Window system which abstracts the hardware and provides the foundation for most 
+graphical user interfaces, like desktop environments or window managers, and 
+their applications.
+
+
+Getting Started
+________________________________________________________________________________
+
+If you are following along from the KISS installation guide, you should already
+have the following packages installed:
+
+* xorg-server
+* xf86-input-libinput
+* liberation-fonts (optional)
+
+In order to start the X window system server, you will also need to install
+and configure _ONE_ of the following packages:
+
+* xinit (@/xorg/xinit)
+* sx (@/xorg/sx)
+
+
+Graphics Drivers
+________________________________________________________________________________
+
+Depending on your graphics card hardware, one of the following graphics card 
+packages should also be installed:
+
+* xf86-video-amdgpu
+* xf86-video-ati
+* xf86-video-intel
+* xf86-video-nouveau
+* xf86-video-vesa
```

Comment:
Should be noted that intel GPU users don't need to install `xf86-video-intel`. The generic modesetting driver (built into Xorg) works really well.

Reply:
added as an additional note (almost word-for-word lol).